lib/icinga/apievents.cpp

```diff
@@ -3,7 +3,6 @@
 #include "icinga/apievents.hpp"
 #include "icinga/service.hpp"
 #include "icinga/notificationcommand.hpp"
-#include "remote/eventqueue.hpp"
```

Comment:
Why are you removing this?

Reply:
I included it in the header file because I needed it there for the parameters and I removed it from here so it wouldn't be imported twice.